src/Microsoft.AspNet.Authentication.Cookies/CookieAppBuilderExtensions.cs

```diff
@@ -20,13 +20,10 @@ public static class CookieAppBuilderExtensions
         /// <param name="configureOptions">Used to configure the options for the middleware</param>
         /// <param name="optionsName">The name of the options class that controls the middleware behavior, null will use the default options</param>
         /// <returns>The original app parameter</returns>
-        public static IApplicationBuilder UseCookieAuthentication([NotNull] this IApplicationBuilder app, Action<CookieAuthenticationOptions> configureOptions = null, string optionsName = "")
+        public static IApplicationBuilder UseCookieAuthentication([NotNull] this IApplicationBuilder app, Action<CookieAuthenticationOptions> configureOptions = null)
```

Comment:
Yeah I need to merge those changes back in from the other dead PR... will update soon